### ISSUE TITLE
How to correct page break if text is larger than cell and page?

### ISSUE THREAD

**Issue by sp00kyDD** [simonbengtsson/jsPDF-AutoTable#195]
Hi!

How can I achieve that a very long text will continue on the next site (page break) and not overflow the border?

Thx!
```
doc.autoTable(columns, rows, {
	theme: "plain",
	margin: { top: 120 },
	styles: {
		font: "helvetica",
		fontStyle: "normal",
		fontSize: 10,
		textColor: [0, 0, 0]
	},
	drawHeaderRow: function () {
		return false;
	},
	drawRow: function (row, data) {
		var currentRow = row;
		if ((currentRow.cells[0].raw != "ID") && (currentRow.cells[0].raw != "Status")) {
			if (currentRow.cells[1].raw != currentRow.cells[2].raw) {
				currentRow.cells[2].styles.fontStyle = "bolditalic";
				currentRow.cells[2].styles.textColor = [255, 0, 0];
			}
		}
	},
	overflow: "linebreak"
});
```
![image](https://cloud.githubusercontent.com/assets/23478349/20308100/c4f28a9c-ab42-11e6-995f-be2dc0d583fd.png)

**Comment by simonbengtsson** [comment]
This is currently a limitation/bug of the plugin. Was looking into it a while ago, but turned out to be quite complicated. Will look into this for the next major version, but will probably not be for a while.

**Comment by sp00kyDD** [comment]
OK thx for the feedback...

**Comment by simonbengtsson** [comment]
Let me know if you find a workaround or start looking into making a fix yourself!

**Comment by sp00kyDD** [comment]
I'll try... do you think **splitTextToSize** could be an option?

**Comment by simonbengtsson** [comment]
I left a TODO on line 416 in `main.js` where the logic should probably take place.

**Comment by simonbengtsson** [comment]
Just fixed this in commit 3d21a3882. Will release the fix in the next version (2.1.1).
